students/jeanruggiero/Lesson03/mailroom.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+import statistics
+
+# This script maintains a databse of donors including name and donation amounts
+
+# Create a data structure that contains the donors an a history of the amounts
+# they have donated
+donors = (\
+('Han Solo', [3468.34, 457, 34.2]),\
+('Luke Skywalker', [5286286.3, 567, 23.5678]),\
+('Chewbacca', [432, 679.4553]),\
+('Princess Leia', [5.3434]),\
+('Bobba Fett, Bounty Hunter', [67]))
```

Comment:
tuple is not the right data structure here, tuple is immutable - instead choose mutable data structure like a `list`:
```
donors = [("John", [100, 200]), ("Amy", [399])]